pilot/pkg/proxy/envoy/v1/testdata/weighted-route-v1alpha2.yaml.golden

```diff
@@ -1,12 +1,14 @@
 hosts:
-  - world
+- world
+gateways:
+- some-gateway
 http:
-  - route:
-    - destination:
-        name: world
-        subset: v0
-      weight: 75
-    - destination:
-        name: world
-        subset: v1
-      weight: 25
+- route:
+  - destination:
+      name: world
+      subset: v0
+    weight: 75
+  - destination:
+      name: world
+      subset: v1
+    weight: 25
```

Comment:
did you run `REFRESH_GOLDEN=true make pilot-test` to regenerate the goldens? May be thats why the weighted route is failing?